dashboard/src/components/MigrateForm/MigrateForm.tsx

```diff
@@ -0,0 +1,270 @@
+  releaseName: string;
+  chartValues: string | null | undefined;
+  chartName: string;
+  chartRepoAuth: {};
+  chartRepoName: string;
+  chartRepoURL: string;
+  repos: IAppRepository[];
+}
+
+interface IMigrationtFormState {
+  releaseName: string;
+  chartValues: string;
+  chartVersion: string;
+  namespace: string;
+  chartName: string;
+  chartRepoName: string;
+  chartRepoURL: string;
+  chartRepoAuth: {};
+  repos: IAppRepository[];
+}
+
+class MigrateForm extends React.Component<IMigrationFormProps, IMigrationtFormState> {
+  public state: IMigrationtFormState = {
+    chartName: this.props.chartName,
+    chartRepoAuth: {},
+    chartRepoName: this.props.chartRepoName,
+    chartRepoURL: "",
+    chartValues: this.props.chartValues || "",
+    chartVersion: this.props.chartVersion,
+    namespace: this.props.namespace,
+    releaseName: this.props.releaseName,
+    repos: this.props.repos,
+  };
+
+  public render() {
+    return (
+      <div>
+        <form className="container padding-b-bigger" onSubmit={this.handleDeploy}>
+          <div className="row">
+            <div className="col-8">{this.props.error && this.renderError()}</div>
+            <div className="col-12">
+              <h2>{this.props.chartID}</h2>
+            </div>
+            <div className="col-12">
+              <p>
+                In order to be able to manage {this.state.releaseName} select the repository it can
+                be retrieved from.
+              </p>
+            </div>
+            <div className="col-8">
+              <div>
+                <label htmlFor="chartRepoName">Chart Repository Name *</label>
+                <select
+                  id="chartRepoName"
+                  onChange={this.handleChartRepoNameChange}
+                  value={this.state.chartRepoName}
+                  required={true}
+                >
+                  {this.state.chartRepoName === "" && <option key="" value="" />}
+                  {this.props.repos.map(r => (
+                    <option key={r.metadata.name} value={r.metadata.name}>
+                      {r.metadata.name}
+                    </option>
+                  ))}
+                </select>
+              </div>
+              <div>
+                <label htmlFor="chartRepoURL">Chart Repository URL</label>
+                <input
+                  id="chartRepoURL"
+                  value={
+                    this.state.chartRepoURL === ""
+                      ? "(Select a repository name)"
+                      : this.state.chartRepoURL
+                  }
+                  required={true}
+                  disabled={true}
+                />
+              </div>
+              <div>
+                <p>
+                  {" "}
+                  * If the repository containing this chart is not in the list add it{" "}
+                  <a href="/config/repos"> here </a>.{" "}
+                </p>
+              </div>
+              <div>
```

Comment:
I would remove everything below this point. None of this is configurable (and doesn't need to be for importing an app). Let's keep this form simple.

BTW, a future improvement for this form would be to search for the chart and version across all the repos and display the possible charts from the different repos it could belong too. We should create an issue for this when this gets merged.